### PR TITLE
refactor: durantion的默认值使用defaultProps

### PR DESCRIPTION
**Pull request by xch1029**
和visibilityHeight保持一致

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     refactor duration default value      |
| 🇨🇳 Chinese |    重构时长的默认值的形式       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
建议去掉 defaultProps，能省一点体积。

File: components/back-top/index.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ const BackTop: React.FC<BackTopProps> = props => {
       </div>
     );
     return (
-      <CSSMotion visible={visible} motionName={`${rootPrefixCls}-fade`} removeOnLeave>
```
Comment (afc163) [inline components/back-top/index.tsx:94]: 为啥要改这个。

Comment (xch1029) [reply components/back-top/index.tsx:94]: removeOnLeave默认值为true，随手删了

**Comment by afc163** [comment]
ping

**Comment by afc163** [comment]
Closing for no response, feel free to create another PR any moment.

**Pull request closed without merging**